src/Elastic/custom/New-AzElasticFilteringTagObject.ps1

```diff
@@ -0,0 +1,53 @@
+
+    # ----------------------------------------------------------------------------------
+    #
+    # Copyright Microsoft Corporation
+    # Licensed under the Apache License, Version 2.0 (the \"License\");
+    # you may not use this file except in compliance with the License.
+    # You may obtain a copy of the License at
+    # http://www.apache.org/licenses/LICENSE-2.0
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an \"AS IS\" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    # See the License for the specific language governing permissions and
+    # limitations under the License.
+    # ----------------------------------------------------------------------------------
+
+    <#
+    .Synopsis
+    Create a in-memory object for FilteringTag
+    .Description
+    Create a in-memory object for FilteringTag
+
+    .Outputs
```

Comment:
Could we use `model-cmdlet` to generate this class?

Reply:
Add ArgumentCompleterAttribute for Action parameter after generated this class use model-cmdlet.